hledger/Hledger/Cli/Commands/Balance.hs

```diff
@@ -306,6 +307,7 @@ balancemode = hledgerCommandMode
    ,flagNone ["percent", "%"] (setboolopt "percent") "express values in percentage of each column's total"
    ,flagNone ["invert"] (setboolopt "invert") "display all amounts with reversed sign"
    ,flagNone ["transpose"] (setboolopt "transpose") "transpose rows and columns"
+   ,flagReq  ["width","w"] (\s opts -> Right $ setopt "width" s opts) "N" "set the maximum width to use for each amount in a balance report"
```

Comment:
I see how it works. In multiperiod balance reports it sets a maximum width for rendering a multicommodity amount. You can set it to be less than or greater than the default, which I believe is 30 characters. I think the idea is that you can set it to a larger width, so that amounts are not elided. The effect is to widen just those columns which have wide amounts, just enough to display the amount without eliding; other columns remain at their usual width. When I tried it I had to make several attempts to find a wide-enough width (I ended up at `-w1000`). The end result was the same as using `--no-elide`, and that flag seems easier. So this new option doesn't seem all that useful in its present form.

Reply:
The main disadvantage of `--no-elide` is that it has other effects: namely it will prevent boring account elision. I would ideally like a way to widen the columns without losing that advantage.

Perhaps a better option is to have the default column width be unbounded, and be able to set a narrower width using `--width` if you desire. We would then be able to reserve ‘--no-elide’ to only control boring parent elision.

Reply:
That said, your subsequent proposals for multi-line balances in multi-period  reports (#1559) would be another way of addressing this problem, greatly reducing the need to widen columns at all. That could either be complementary to this proposal, or replace it in some way.